snippets/emulated-gpio/emulated-gpio.overlay

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&cpuflpr_vpr {
+    status = "okay";
+
+    egpio: gpio {
+        compatible = "nordic,nrf-egpio";
+        gpio-controller;
+        #gpio-cells = <0x2>;
+        ngpios = <0x10>;
+        status = "disabled";
+        port = <0x2>;
+    };
+};
+
+&cpuapp_vevif_tx {
+    status = "okay";
+};
```

Comment:
This should also be removed from the main snippet file. It should only be in the snippet for board.